ledger/src/sigverify_shreds.rs

```diff
@@ -272,7 +275,7 @@ fn sign_shred_cpu(keypair: &Keypair, packet: &mut Packet) {
         packet.meta.size >= sig.end,
         "packet is not large enough for a signature"
     );
-    let signature = keypair.sign_message(&packet.data()[msg]);
+    let signature = keypair.sign_message(packet.data(msg).unwrap_or_default());
```

Comment:
If there's no message, this should bail before signing

Reply:
The node only signs its own shreds, so if this is no message then it is an invalid shred. So that means this should never happen.
I can change this to `.unwrap()` to be more noisy if something goes wrong, similar to `assert` just above it.
Would that make sense?

Reply:
Yeah that works for me

Reply:
done